### PR TITLE
Update of panel for inset attribute

### PR DESCRIPTION
**Pull request by PtitBen**

### CODE REVIEW

**Review by RobTobias123 — COMMENTED** [review]
Needs the initail HTML example removing so we only have the standard and inset not a repeat of the standard

**Comment by PtitBen** [comment]
> Needs the initail HTML example removing so we only have the standard and inset not a repeat of the standard

Fixed already ;)

File: src/content/docs/components/ns-panel.mdx [hunk 1]
```diff
@@ -19,25 +19,51 @@ import Figma from '@components/figma.astro';
...
 
-<ExampleViewer type="standard" />
+<Tabs>
+  <TabItem label="Standard">
+    The standard panel, goes from the browser's edge to edge
+    <ExampleViewer type="standard" />
+  </TabItem>
+  <TabItem label="Inset">
+   To break the page's rythm and flow, the inset panel is contained and has margins on each side.
+    <ExampleViewer type="standard" />
```
Comment (RobTobias123) [inline src/content/docs/components/ns-panel.mdx:42]:
```suggestion
    <ExampleViewer type="inset" />
```

Comment (RobTobias123) [reply src/content/docs/components/ns-panel.mdx:42]: It was still showing the standard in the example even though I could see the inset.html had been added.

**Review by RobTobias123 — APPROVED** [review]
Looks good to me

**Review by andij — APPROVED** [review]

**Pull request merged**